content/docs/use-cases/sharing-resources-efficiently.md

```diff
@@ -0,0 +1,79 @@
+# Sharing Resources Efficiently
+
+Data science teams need to handle large files, rotate the use og special
+processors, and minimize data transfers. This involves provisioning and managing
+resources such as massive on-prem data stores and powerful servers, which can be
+expensive and time consuming.
+
+![](/img/shared-server.png) _Data store shared by DVC projects_
+
+DVC projects support different ways to optimize resource utilization in order to
+minimize cost and complexity. This can make the difference, for example when:
+
+- Multiple users work on the same shared server, or there's a single computing
+  environment to run experiments.
+- GPU time gets distributed among people or processes for training machine
+  learning models.
+- There's a centralized data storage unit or cluster.
+
+## Shared Storage
```

Comment:
let's do the whole page w/o subsections first (and ideally w/o them at the end)

Reply:
OK